R/module_teal.R

```diff
@@ -103,6 +102,39 @@ srv_teal <- function(id, data, modules, filter = teal_slices()) {
       shinyjs::showLog()
     }
 
+    insertUI(
```

Comment:
why to `insertUI` when elements don't use any objects created in the server. I think this can be moved to UI directly